src/docs/concepts-development/translations/index.md

```diff
@@ -0,0 +1,192 @@
+pluralization in a text. The ICU format makes it possible to define
+different wordings for singular and plural options.
+
+```js
+/* When you use plural in the translation string, you will need to specify
+- the variable determining which option to use (here: 'count')
+- the pattern we are following (here: 'plural')
+- the options matching each alternative you want to specify (here: 'one' – there could be several options specified)
+- an 'other' option that gets used when none of the specified alternatives matches
+*/
+const translations = {
+  en: {
+    "ManageListingsPage.youHaveListings": "You have {count} {count, plural, one {listing} other {listings}}",
+  }
+}
+...
+  // When we pass 'count' as a value, the formatting determines which translation option, 'one' or the default,
+  // is used in the final message string.
+  const heading = (
+    <FormattedMessage
+      id="ManageListingsPage.youHaveListings"
+      values={{ count: pagination.totalItems }}
+    />
+
+  // If pagination.totalItems === 1, the message reads "You have 1 listing" / "Tienes 1 anuncio".
+  // For any other number, including zero, the message reads "You have X listings" / "Tienes X anuncios".
+)
+```
+
+Since different languages have different pluralization rules,
+pluralization is defined per language. You can see the full list of
+pluralization arguments (`zero`, `one`, `two`, `few` etc.) in the
+[ICU syntax documentation](https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format).
+
+### select
+
+In addition to pluralization options, you can build logic to the
+translation strings using
+[select formatting](https://formatjs.io/docs/core-concepts/icu-syntax/#select-format).
+
+```js
+/* When you use `select` in the translation string, you will need to specify
+- the variable determining which option to use (here: 'actor')
+- the pattern we are following (here: 'select')
+- the options matching each alternative you want to specify (here: 'you' – there could be several options specified)
+- an 'other' option that gets used when none of the specified alternatives matches
+*/
+const translations = {
+  en: {
+    "TransactionPage.bookingAccepted": "{actor, select, you {You accepted the booking request.} other {{otherUsersName} accepted the booking request.}}",
+  }
+}
+...
+// actor: 'you', 'system', 'operator', or display name of the other party
+const actor = 'you';
+
+// For { actor: 'you', otherUsersName: 'Riley' }, the message will read "You accepted the booking request.".
+// For any other values of 'actor', the message would read "Riley accepted the booking request."
+const bookingAccepted = intl.formatMessage(
+  { id="TransactionPage.bookingAccepted" },
+  { actor, otherUsersName }
+);
+
+```
+
+You can use select for cases where you have a predetermined list of
+options you will encounter that require different translation strings.
+
+## Can I have a multilanguage marketplace?
+
+- Technically, having several translation files does enable using a
```

Comment:
"Technically" sounds like it's not easily possible :) Maybe remove it?

Reply:
I guess this is not the final presentation of this chapter. But probably it makes sense to remove "technically" away and just point out that _"currently, Flex doesn't support more than 1 translation asset file through Console"_.